DigitalLearningSolutions.Data.Tests/Services/LogoServiceTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    internal class LogoServiceTests
+    {
+        private LogoService logoService;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            logoService = new LogoService(connection);
+        }
+
+        [Test]
+        public void Get_Logo_Should_Return_Centre_logo()
+        {
+            // When
+            var result = logoService.GetLogo(25, 1176);
+
+            // Then
+            result.Should().NotBeNull();
+            result.LogoUrl.Should().NotBeNullOrEmpty();
+            result.Name.Should().Be("Royal Cornwall Hospitals NHS Trust");
+        }
+
+        [Test]
+        public void Get_Logo_Should_Return_Centre_logo_When_CustomisationId_Is_Null()
+        {
+            // When
+            var result = logoService.GetLogo(25, null);
+
+            // Then
+            result.Should().NotBeNull();
+            result.LogoUrl.Should().NotBeNullOrEmpty();
+            result.Name.Should().Be("Royal Cornwall Hospitals NHS Trust");
+        }
+
+        [Test]
+        public void GetLogo_Should_Return_Brand_Logo_When_No_Centre_Logo()
+        {
+            // When
+            var result = logoService.GetLogo(3, 7793);
+
+            // Then
+            result.Should().NotBeNull();
+            result.LogoUrl.Should().NotBeNullOrEmpty();
+            result.Name.Should().Be("IT Skills Pathway");
+        }
+
+        [Test]
+        public void GetLogo_Should_Return_Null_LogoUrl_When_No_Logo_Is_Available()
+        {
+            // When
+            var result = logoService.GetLogo(3, 100);
+
+            // Then
+            result.LogoUrl.Should().BeNull();

```

Comment:
It probably makes more sense for the whole logo object to be null in this case? 🙂

Reply:
Yeah I agree. Fixed when I refactored the query in 60805a3 🙂